setup.py

```diff
@@ -48,7 +48,7 @@ def get_packages(package):
     "uvloop>=0.14.0 ;" + env_marker,
 ]
 
-extras_require = {"watchgodreload": ["watchgod>=0.6,<0.7"]}
+extras_require = {"watchgodreload": ["watchgod>=0.6,<0.7"], "yamllogconfig": ["PyYAML>=5.1"]}
```

Comment:
This PR also adds items to the `extras_require` list: #298. There's a fair amount of discussion there trying to come up with the best names for the optional features. As this will become part of the public interface and troublesome to update once it's live, it might make sense to omit this line for now until the best name for this feature is agreed upon. This wouldn't conflict with the changes in #298, but there they appear to be trying to name the extras as categories as opposed to features to enable.

Reply:
yep I'd prefer @tomchristie to decide on this, not sure we should pile on extras_require for every little tweak, even if we did it already with the reloader

Reply:
I backed this change out